### ISSUE TITLE
The 'My Catalog' page is missing an H1 level heading

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#1632]
### Description
Headings serve as an important navigation mechanism for users of accessible technology. Each page should have a single h1 level heading consistent with the overall topic of the page, followed by additional headings nested in hierarchical order where appropriate. The 'My Catalog' page is missing an h1 level heading that describes the purpose of the page. Instead the page starts with a postbox which features a h4 level heading and follows this with an h2 element. To fix simply, the h2 can be turned into an h1 and moved before the postbox. See https://github.com/pressbooks/pressbooks/blob/a93df1e7297b8ad0a4d9339b8d735c2a1ebdc06b/inc/admin/class-catalog-list-table.php#L460-L480
